es-core/src/components/VideoComponent.cpp

```diff
@@ -221,6 +212,15 @@ void VideoComponent::applyTheme(const std::shared_ptr<ThemeData>& theme, const s
 	if (elem->has("showSnapshotDelay"))
 		mConfig.showSnapshotDelay = elem->get<bool>("showSnapshotDelay");
 
+	setRotation(0.0);
```

Comment:
Unsure if we're saying the same thing - my question is whether we need to explicitly reset the rotation and size, given that two lines below we're explicitly setting it, should the elements have those attributes set.

It doesn't do any harm, really - just wondering if it's not redundant.